app/controllers/github_controller.rb

```diff
@@ -173,6 +173,7 @@ def update_deploy_to_master
     # See https://developer.github.com/v3/activity/events/types/#webhook-payload-example-26
     # for what’s in `params`
     ActionCable.server.broadcast 'smokedetector_messages', deploy_updated: params
+    ApiChannel.broadcast_to 'api_deploy_update', event_type: 'update', event_class: 'Ref', object: params
```

Comment:
`channel` should match `event_class.pluralize.underscore` - not sure if it _actually_ makes a difference, but good to be consistent